javascript-loops-practice/get-indexes.js

```diff
@@ -0,0 +1,10 @@
+/* exported getIndexes */
+function getIndexes(array) {
+  var stringArray = [];
+  var indexesArray = [];
+  for(var i = 0; i < array.length; i++) {
+    stringArray.push(array[i]);
```

Comment:
What is the purpose of `stringArray`? You don't use it for anything, it should be deleted.